### ISSUE TITLE
Don't mark rooms as "unread" for m.notice messages

### ISSUE THREAD

**Issue by clokep** [vector-im/element-meta#2079]
#### What would you like to do?

Currently whenever a renderable event is received it marks the room as "unread" (the code calls this "bold"). (There's some [small docs around this](https://github.com/matrix-org/matrix-react-sdk/blob/develop/docs/room-list-store.md#list-ordering-algorithm-importance).) This applies to messages including the [`m.notice` `msgtype`](https://spec.matrix.org/v1.8/client-server-api/#mnotice), which is used for "automated clients".

There are some rooms I'm in that have *a lot* of webhooks setup via [hookshot](https://github.com/matrix-org/matrix-hookshot/), this causes those rooms to constantly be marked as unread, even though there's really no action for me to take.

I'd like to be able to disable marking of rooms as unread for `m.notice` messages.

#### Why would you like to do it?

I would like to do this because it is wasting my time checking rooms that don't have any actionable messages in them.

#### How would you like to achieve it?

Either stop marking rooms as unread for `m.notice` messages by default or via a configuration option.

### Have you considered any alternatives?

The other option I consider is blocking the hookshot user, but this will cause some context in the rooms to be lost.

### Additional context

Note that I have "Messages sent by bots" set to disabled in my notification settings and in general only get notifications for pings.

**Comment by Johennes** [comment]
We'll probably need @daniellekirkwood to weigh in here. I'm not sure if we should exclude notices by default. Whether or not their actionable is probably subjective and also depends on the type of notice.

**Comment by clokep** [comment]
> Whether or not their actionable is probably subjective and also depends on the type of notice.

Yes, I'm not sure how to overcome that though. I wonder if an option would be that if I *muted* the user then that also would apply to unread-ness. (Muting being different than ignoring; although I don't think Element Web has UI for muting.)
